lib/ssl/src/inet_tls_dist.erl

```diff
@@ -897,6 +972,39 @@ verify_fun(Value) ->
 	    error(malformed_ssl_dist_opt, [Value])
     end.
 
+set_ktls(#{
+    socket := Socket,
+    tls_version := {3, 4},
+    cipher_suite := ?TLS_AES_256_GCM_SHA384,
+    socket_options := #socket_options{
+        mode = _Mode,
+        packet = Packet,
+        packet_size = PacketSize,
+        header = Header,
+        active = Active
+    },
+    write_state := #cipher_state{
+        iv = <<WriteSalt:4/bytes, WriteIV:8/bytes>>, key = WriteKey
+    },
+    write_seq := WriteSeq,
+    read_state := #cipher_state{
+        iv = <<ReadSalt:4/bytes, ReadIV:8/bytes>>, key = ReadKey
+    },
+    read_seq := ReadSeq
+}) ->
+    % SOL_TCP = 6, TCP_ULP = 31
```

Comment:
There should be a test for `os:version()` being `{unix,linux`, and I guess also `os:version()` being better than some version.  Nginx claims that 4.13 supports kTLS but they recommend 5.2, don't know why.  So test for `os:version()` better than `{5,2,0}`?
Otherwise we go and set unknown raw socket options on unknown platforms.